setup.py

```diff
@@ -17,10 +17,10 @@
         'cadscript>=0.5.2',
         'exsource-tools',
         'cq-cli @ git+https://github.com/CadQuery/cq-cli.git',
-        'gitbuilding>=0.0.6',
+        'gitbuilding==0.15.0a4',
```

Comment:
Isn't this the GitBuilding version that caused the problem the other day?

Reply:
No it was alpha 2 with the bug, but we need the version 15 alphas for the features added during the NLNet project. I missed what version had been set when reviewing #92 . The `>=0.0.6` was the setting for exsource that GitBuilding had written as `==0.0.6`. With this configuration it should all work.